.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -32,7 +32,9 @@ body:
     id: device
     attributes:
       label: Device
-      description: The device exhibiting this bug.
+      description: |
+      The device exhibiting this bug (if unsure, use command below).
+      ```cat /tmp/sysinfo/model```
```

```suggestion
      description: |
        The device exhibiting this bug (if unsure, use command below).
        ```cat /tmp/sysinfo/model```
```

The yaml is not correct and require additional tab

---

Fixed